### ISSUE TITLE
Gradually increase LFC quota without taking working set into consideration

### ISSUE THREAD

**Issue by stradig** [neondatabase/autoscaling#871]
To improve performance of autoscaled computes @Bodobolero's tests showed that the LFC quota can be a deciding factor for the performance. As an experiment we want to gradually increase the LFC quote and see what benefits we are getting from that and in what kinds of problems we are running into.

**Comment by skyzh** [comment]
ref issue on the compute side https://github.com/neondatabase/neon/issues/6919

**Comment by stradig** [comment]
@skyzh Is this finished or are you planning further changes?

**Comment by skyzh** [comment]
Planning on remove the limit on vm-builder. Plan to do it after the swap config change gets released so that the disk usage metrics can be observed without interference of other factors.

**Comment by skyzh** [comment]
vm-monitor changes: https://github.com/neondatabase/neon/pull/8066

**Comment by Bodobolero** [comment]
@skyzh do you plan further size increases for LFC ?

**Comment by skyzh** [comment]
yep, I'm still working on https://github.com/neondatabase/neon/pull/8066. The current plan is to pass active_ratio over compute startup config so that we can roll it out safely.

**Comment by stradig** [comment]
@skyzh Are you intending to do more about this or should this be closed. If we continue with this, this should be assigned to some other project, or?
